photon-lib/src/main/java/org/photonvision/estimation/VisionEstimation.java

```diff
@@ -103,7 +103,7 @@ public static PNPResults estimateCamPosePNP(
         Point[] points = OpenCVHelp.cornersToPoints(corners);
 
         // single-tag pnp
-        if (visTags.size() == 1) {
+        if (knownTags.size() == 1) {
```

Comment:
what happens if knownTags.size = 0?

Reply:
```java
if (knownTags.size() == 0 || corners.size() == 0 || corners.size() % 4 != 0) {
    return new PNPResults();
}
```
not involved in this but I assume that this (from earlier) addresses that already

Reply:
Ah, yep. That's my fault for not actually RTFM.